### PR TITLE
Grade gut

### PR DESCRIPTION
**Pull request by markolkc**

### CODE REVIEW

File: create_db.sql [hunk 1]
```diff
@@ -104,3 +101,100 @@ CREATE TABLE Goals (
...
+
+INSERT INTO Game VALUES (3, 2, '{"gameday": 1, "gamedate": "2023-05-11"}');
+INSERT INTO Game VALUES (1, 4, '{"gameday": 2, "gamedate": "2023-05-12"}');
+INSERT INTO Game VALUES (6, 5, '{"gameday": 3, "gamedate": "2023-05-13"}');
+INSERT INTO Game VALUES (7, 9, '{"gameday": 4, "gamedate": "2023-05-14"}');
+INSERT INTO Game VALUES (10, 8, '{"gameday": 5, "gamedate": "2023-05-15"}');
+INSERT INTO Game VALUES (11, 12, '{"gameday": 6, "gamedate": "2023-05-16"}');
+
+
+SELECT
```
Comment (schletz) [inline create_db.sql:187]: Remove this select (I think this was for debugging)

**Review by schletz — CHANGES_REQUESTED** [review]
JSON Schema missing, see https://github.com/schletz/Dbi2Sem/blob/master/projekt_sommersemester.md#bewertung-befriedigend:

> Ergänzung des Modelles um ein Entity – Attribute – Value Modell (siehe https://github.com/schletz/Dbi2Sem/tree/master/34_EntityAttribute). Es kann wahlweise mit Entities modelliert oder als JSON Spalte angegeben werden. Bei einer JSON Spalte ist ein JSON Schema (https://json-schema.org/learn/getting-started-step-by-step#defining-the-properties) zu definieren.

Write your schema in _game_value_schema.json_

**Comment by markolkc** [comment]
schema and data for the schema done; sql updated

**Review by schletz — CHANGES_REQUESTED** [review]
Your INSERT Statements
```sql
INSERT INTO Game VALUES (3, 2, '{"gameday": 1, "gamedate": "2023-05-11"}');
INSERT INTO Game VALUES (1, 4, '{"gameday": 2, "gamedate": "2023-05-12"}');
INSERT INTO Game VALUES (6, 5, '{"gameday": 3, "gamedate": "2023-05-13"}');
INSERT INTO Game VALUES (7, 9, '{"gameday": 4, "gamedate": "2023-05-14"}');
INSERT INTO Game VALUES (10, 8, '{"gameday": 5, "gamedate": "2023-05-15"}');
INSERT INTO Game VALUES (11, 12, '{"gameday": 6, "gamedate": "2023-05-16"}');
```
does not validate against your schema in *game_value_schema.json* (*Required properties are missing from object: Id, HomeTeam, AwayTeam, GameDate.*)

Your data in *data_game_value_schema.json* does not validate against your schema (check each object in your array).

Check https://www.jsonschemavalidator.net if your json is valid.

**Review by schletz — CHANGES_REQUESTED** [review]
> Your view *GameView* doesn't work anymore because JSON keys are case sensitive (returns always NULL for gameday and gamedate).

My fault, you corrected this.

**Review by schletz — APPROVED** [review]

**Comment by schletz** [comment]
Good work, you can start with branch _sehr gut_. 

For sehr gut you can choose: You can map a view to ef core (as described in https://github.com/schletz/Dbi2Sem/blob/master/projekt_sommersemester.md#bewertung-sehr-gut) or you can do the following, if you are interested in.

Create a .NET 7 (not 6) Program and import your model with ef power tools (or another scaffolder). Use inheritance for your vehicle types and map the discriminator to your Discriminator column.

You can map your JSON column to a C# class, see https://www.youtube.com/watch?v=Nqd3snxmILE

Create a unittest to add some vehicle with some additional infos and try to query vehicles with specified attributes.

**Pull request merged**